### ISSUE TITLE
Problem with Accessibility

### ISSUE THREAD

**Issue by piovac** [ViennaRSS/vienna-rss#416]
Vienna does not seem to register correctly with the Accessibility System Preference in Maverick. As a consequence it is not possible to write applescript scripts that access it's menu items with System Events.

I wrote few scripts (for instance to print the news or save them in a PDF file) that work just fine if launched with Applescript Editor but fail if run from the application script menu. In such a situation Vienna should ask to be added to the Accessibility list, but it does not. Even adding Vienna manually to the Accessibility list does not solve the problem. 

I tried to run the same scripts in other applications like Skim and they work as expected both in th editor and the app (when ran in the app the first time Skim ask to be added to the Accessibility list as it should).

This should be fixed because it impairs important work flows, like the possibility to archive automatically PDF version of the news to BibDesk for offline reading and annotation.

**Comment by josh64x2** [comment]
Related to issue #387

**Comment by josh64x2** [comment]
@piovac are you able to share one of the scripts that doesn't work so I can use it for testing please?

Thanks!

**Comment by piovac** [comment]
Josh, Below is the code of the "Save PDF to Desktop.scpt" script. The codes work when run inside the Applescript editor but not when run in the Vienna menu. See the comment on the value of the varialble "c". Similar code works for Skim (even if it need to be modified because Skim has a popup sheets instead of a popup dialog box).

set maxloops to 5
set pdfSavePath to "/Users/piovac/Desktop"
set pdfFileName to "tmp000.pdf"

tell application "System Events"
	tell process "Vienna"
		set frontmost to true
		-- Following two commented lines are for debug purpose:
		-- "c" should be equal to "1", but when run inside the Vienna app it returns "0",
		-- while when run inside Applescript editor it returns "1" as expected
		--set c to (count menu bars) as text
		--display alert c
		click menu item "Print…" of menu 1 of menu bar item "File" of menu bar 1
		repeat until window "Print" exists
		end repeat
		-- Click the PDF menu button
		click menu button "PDF" of window "Print"
		-- Make sure the menu is up
		repeat until exists menu item "Save as PDF…" of menu 1 of menu button "PDF" of window "Print"
		end repeat
		-- Select the "Save as PDF" menu item
		click menu item "Save as PDF…" of menu 1 of menu button "PDF" of window "Print" -- Save as PDF...
		-- Make sure the save dialog is visible
		repeat until exists sheet 1 of window "Print"
		end repeat
		-- Press command+shift+g to show the "Go" drop down sheet
		keystroke "g" using {command down, shift down}
		-- Set the location field to our pdfSavePath
		set value of text field 1 of sheet 1 of sheet 1 of window "Print" to pdfSavePath
		-- Now click the Go button
		click button "Go" of sheet 1 of sheet 1 of window "Print"
		-- Now that we are in our desired folder, set the file name and save
		set value of text field of sheet 1 of window "Print" to pdfFileName
		-- And save it!		
		click button "Save" of sheet 1 of window "Print"
	end tell
end tell
